client.go

```diff
@@ -489,17 +489,32 @@ func (c *Client) websocket(operation string, secret string) (*websocket.Conn, er
 }
 
 func (c *Client) url(elem ...string) string {
+	// Normalize the URL
 	path := strings.Join(elem, "/")
+	entries := []string{}
```

Comment:
That's because strings.Replace("///", "//", "/", -1) gives you "//" instead of "/".